docs/gathered-data.md

```diff
@@ -549,6 +549,24 @@ Response see:
   - 4.8+
 
 
+## Machine
+
+collects Machine information
+
+The Kubernetes api:
+
+	https://github.com/openshift/api/blob/master/machine/v1beta1/types_machine.go
+
+Response see:
+
+	https://docs.openshift.com/container-platform/4.12/rest_api/machine_apis/machine-machine-openshift-io-v1beta1.html
+
+* Location in archive: machines/
```

Comment:
I guess that you mean `machines/{namespace}/{name}.json`?

Reply:
Yes good catch. This should go to `config/machines` probably....note that machinesets extra dir was a mistake, we haven't fixed yet.

Reply:
config/machines, in case of no namespace it just saves the record there